### PR TITLE
User settings work again (minus username changing).

### PR DESCRIPTION
**Pull request by tiffz**
It was bothering me that these didn't work. I made some minor changes to forms to work with this as well.

### CODE REVIEW

File: saylua/models/user.py [hunk 1]
```diff
@@ -95,38 +90,6 @@ def hash_password(cls, password, salt=None):
     def check_password(cls, user, password):
         return cls.hash_password(password, user.phash) == user.phash
 
-    # @classmethod
-    # @ndb.transactional(xg=True)
-    # def transfer_currency(cls, from_key, to_key, cc=0, ss=0):
```
Comment (NoiSek) [inline saylua/models/user.py]: Why remove these? They will need to be re-implemented in the future as far as I understand it.

File: saylua/modules/users/templates/settings/details.html [hunk 2]
```diff
@@ -1,48 +1,18 @@
...
-      <th class="label">
-        {{ form.bio.label }}
-      </th>
-      <td>
-        {{ form.bio(placeholder="Write a bit about yourself for others to see. ") }}
-      </td>
-    </tr>
+<form method="post" class="validated-form">
+  <table class="form-table">
+    {{ form_row(form.status, tip="Your status is a bit of text displayed next to your name.") }}
```
Comment (NoiSek) [inline saylua/modules/users/templates/settings/details.html]: neat.

**Review by LikeMyBread — APPROVED** [review]
Looks good to me! Glad to have this working again.

File: saylua/utils/form/widgets.py [hunk 3]
```diff
@@ -8,7 +8,7 @@ def generateCustomKwargs(field, error_class, kwargs):
         c = kwargs.pop('class', '') or kwargs.pop('class_', '')
         kwargs['class'] = '%s %s' % (error_class, c)
 
-    if 'placeholder' not in kwargs:
+    if 'placeholder' not in kwargs or not kwargs['placeholder']:
```
Comment (NoiSek) [inline saylua/utils/form/widgets.py]: If kwargs is a real dictionary, you can just do:

```python
if not kwargs.get('placeholder'):
  [...]
```

**Pull request merged**